plan_tools/__init__.py

```diff
@@ -1,2 +1,2 @@
 NAME = 'PLAN-Tools'
-VERSION = 0.5
+VERSION = 0.6
```

Comment:
Bump version number, I see this starting to stabilize once I get these specific issues worked out.